client/src/stores/historyStore.ts

```diff
@@ -14,7 +14,9 @@ import {
     updateHistoryFields,
 } from "@/stores/services/history.services";
 
-export type HistorySummary = components["schemas"]["HistorySummary"];
+export type HistorySummary = components["schemas"]["HistorySummary"] & {
+    update_time: string;
```

Comment:
This looks like should be part of HistorySummary, probably got moved to the summary view after creating the schema model, good catch!